alts/build.gradle

```diff
@@ -18,6 +18,7 @@ buildscript {
 dependencies {
     compile project(':grpc-core'),
             project(':grpc-netty'),
+            project(':grpc-grpclb'),
```

Comment:
This should ideally be in `runtime`. That's a different type a dependency like `testCompile` is a different type.